server/py/framework/utils/clients/discovery.py

```diff
@@ -96,6 +96,7 @@ def _resolve_service_method_routes(
             "put": [],
             "delete": [],
             "patch": [],
+            "head": [],
```

Comment:
Why do we need that here? I haven't seen a request type like that in our code

Reply:
needed for docs endpoint